source/neuropod/multiprocess/tensor_utils.hh

```diff
@@ -27,16 +27,16 @@ namespace neuropod
 // This is useful for serialization and to wrap and/or copy tensors between backends.
 // For example, if you had a TorchNeuropodTensor and you wanted to get a tensor compatible
 // with `allocator` without making a copy, you could use this function
-std::shared_ptr<NeuropodTensor> wrap_existing_tensor(NeuropodTensorAllocator &       allocator,
-                                                     std::shared_ptr<NeuropodTensor> tensor)
+static std::shared_ptr<NeuropodTensor> wrap_existing_tensor(NeuropodTensorAllocator &       allocator,
```

Comment:
Interesting, did compiler require it to be static? In C++, the same can be achieved with anonymous namespace

namespace
{

// functions here
}